src/main/java/emissary/command/BaseCommand.java

```diff
@@ -70,73 +81,56 @@ protected void overrideFlavor(String flavor) {
         System.setProperty(ConfigUtil.CONFIG_FLAVOR_PROPERTY, getFlavor());
     }
 
-    @Parameter(names = {"--binDir"}, description = "emissary bin dir, defaults to <projectBase>/bin")
-    private Path binDir;
-
     public Path getBinDir() {
         if (binDir == null) {
-            return Paths.get(getProjectBase().toAbsolutePath().toString() + "/bin");
+            return Paths.get(getProjectBase().toAbsolutePath() + "/bin");
```

Comment:
```suggestion
            return getProjectBase().toAbsolutePath().resolve("bin");
```

(And for other instances of this in the file too)